variant-analysis-workflow.yml

```diff
@@ -0,0 +1,241 @@
+name: ${{ github.event.inputs.workflow_name }}
+
+env:
+  CODEQL_ENABLE_EXPERIMENTAL_FEATURES_SWIFT: true
+
+on: dynamic
```

Comment:
Is this missing the `TEST_PAT` stuff intentionally?  How do you see tests working once the other workflow file is gone?

Reply:
Thanks for raising this. I think you've pointed out some tech debt and the `TEST_PAT` is actually not used, maybe? I certainly can't find any place where it is set from, so I think it's always empty and unused.

Unless we find out where it's set from, I'm going to remove references to it from the new workflow file.